When running with install folder, no need to add additional paths
diff --git a/test/rubygems/test_require.rb b/test/rubygems/test_require.rb
index 5b826d052b..1293cd41fe 100644
--- a/test/rubygems/test_require.rb
+++ b/test/rubygems/test_require.rb
@@ -451,7 +451,7 @@ def test_realworld_default_gem
       require "json"
       puts Gem.loaded_specs["json"]
     RUBY
-    output = Gem::Util.popen(*ruby_with_rubygems_in_load_path, "-e", cmd).strip
+    output = Gem::Util.popen('ruby', "-e", cmd).strip
     assert $?.success?
     refute_empty output
   end
@@ -471,7 +471,7 @@ def test_realworld_upgraded_default_gem
     RUBY
     File.write(path, code)
 
-    output = Gem::Util.popen({ 'GEM_HOME' => @gemhome }, *ruby_with_rubygems_in_load_path, path).strip
+    output = Gem::Util.popen({ 'GEM_HOME' => @gemhome }, 'ruby', path).strip
     assert $?.success?
     refute_empty output
     assert_equal "999.99.9", output.lines[0].chomp
